src/components/LoanCards/ExpandableLoanCard/ExpandableLoanCardExpanded.vue

```diff
@@ -87,6 +111,16 @@ export default {
 		resumeHover() {
 			this.hoverIsActive = true;
 		},
+		arrowsVisible() {
+			/*
+			We are using the lack of CSS hover support to gate the visibility of the arrows. That doesn't sync with
+			usingTouch unfortunately.
+			*/
```

Comment:
So `usingTouch` was working inconsistently for me. It definitely didn't sync up with our hiding the arrows since that was CSS based `@media (hover: none)`. This discrepancy caused broken app states so I needed to use this.